public/js/ajax-city-handler.js

```diff
@@ -0,0 +1,172 @@
+$.ajaxSetup({
+    headers: {
+        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content'),
+    }
+});
+
+$('form').submit(function(event){
+    event.preventDefault();
+})
+
+const createCity = () => {
+    const page = currentPage();
+    $.ajax({
+        url: `api/cities/?page=${page}`,
+        method: 'POST',
+        data: $('#add-city-form').serialize(),
+        cache: false,
+        success:function(response)
+        {
+            document.forms["add-city-form"].reset();
+            $('#name-error').text('');
+            loadTable();
+        },
+        error: function(xhr) {
+            let errors = xhr.responseJSON.errors;
+            console.error(xhr.responseText)
+        }
+    });
+}
+
+const deleteCity = (cityId) => {
+    const page = currentPage();
+    $.ajax({
+        url: `/api/cities/${cityId}?page=${page}`,
+        method: 'DELETE',
+        cache: false,
+        success:function(response)
+        {
+            loadTable();
+        },
+        error: function(xhr) {
+            let errors = xhr.responseJSON.errors;
+            console.error(xhr.responseText)
+        }
+    });
+}
+
+const updateCity = (cityId) => {
+    const page =  currentPage();
+    $.ajax({
+        url: `/api/cities/${cityId}?page=${page}`,
+        method: 'PUT',
+        data: $('#cities-update-form').serialize(),
+        cache: false,
+        success:function(response)
+        {
+            loadTable();
+        },
+        error: function(xhr) {
+            let errors = xhr.responseJSON.errors;
+            console.error(xhr.responseText)
+        }
+    });
+}
+
+const loadTable = () => {
+    const page = currentPage();
+
+    $.ajax({
+        url: `/api/cities?page=${page}`,
+        method: 'GET',
+        success:function(response)
+        {
+            const cities = response.data;
+            const links = response.links;
+
+            $('#table-body').html(renderTable(cities))
+            $('#pagination-links').html(renderLinks(response));
+        },
+        error: function(xhr) {
+            let errors = xhr.responseJSON.errors;
+            console.error(xhr.responseText);
+        }
+    });
+}
+
+const renderTable = (cities) => {
+    tableBody = '';
```

Comment:
```suggestion
    let tableBody = '';
```
`tableBody = ''` equals to `var tableBody = ''`, and as you might remember, we shouldn't use `var`. In this case, we must declare the variable as `let` because you will mutate it.